test/extended/util/annotate/rules.go

```diff
@@ -113,6 +113,11 @@ var (
 		"[Flaky]": {
 			`Job should run a job to completion when tasks sometimes fail and are not locally restarted`, // seems flaky, also may require too many resources
 			`openshift mongodb replication creating from a template`,                                     // flaking on deployment
+			// Skip the following 4 tests until https://bugzilla.redhat.com/show_bug.cgi?id=1822755 is fixed.
```

Comment:
IMO, we should even go further and make all Network Granular Checks as flaky until https://bugzilla.redhat.com/show_bug.cgi?id=1794714 is resolved, assume they're all caused by the same issue. The search for [Networking Granular Checks over the past 7 days](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520) currently shows 334 failed jobs.

- [Networking Granular Checks: Services should function for node-Service](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+function+for+node-Service&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 172 failures
- [Networking Granular Checks: Services should function for pod-Service](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+function+for+pod-Service&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 150 failures

Some more granular checks failures not captured here:
- [Networking Granular Checks: Services should function for endpoint-Service](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+function+for+endpoint-Service&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 124 failures
- [Networking Granular Checks: Pods should function for intra-pod communication](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Pods+should+function+for+intra-pod+communication&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 112 failures
- [Networking Granular Checks: Pods should function for node-pod communication](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Pods+should+function+for+node-pod+communication&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 107 failures
- [Networking Granular Checks: Services should be able to handle large requests](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+be+able+to+handle+large+requests&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 68 failures
- [Networking Granular Checks: Services should function for client IP based session affinity](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+function+for+client+IP+based+session+affinity&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 76 failures
- [Networking Granular Checks: Services should update endpoints](https://search.svc.ci.openshift.org/?search=Networking+Granular+Checks%3A+Services+should+update+endpoints&maxAge=168h&context=1&type=junit&name=&maxMatches=5&maxBytes=20971520): 120 failures

Reply:
@mandre thanks for catching the additional test failures. I pushed a new commit that covers most of the failed tests that you reference. The other failed tests are covered by [these rules](https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go#L184-L199).